ingot-types/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub trait ToOwnedPacket: NextLayer {
 
     /// Converts a borrowed view of a header into an owned version, possibly
     /// reparsing to do so with the aid of `hint`.
-    fn to_owned(&self, hint: Option<Self::Denom>) -> ParseResult<Self::Target>;
+    fn to_owned(&self, hint: Option<Self::Hint>) -> ParseResult<Self::Target>;
```

Comment:
Yeah, this should have been `Hint` -- I think this mistake fell out of (or was allowed by) this machinery only being really necessary for `Repeated`.